packages/eslint-config/index.cjs

```diff
@@ -4,29 +4,20 @@ module.exports = {
     'turbo',
     'eslint:recommended',
     'plugin:@typescript-eslint/recommended',
-    'plugin:prettier/recommended',
```

Comment:
removed prettier eslint plugin 

https://www.joshuakgoldberg.com/blog/you-probably-dont-need-eslint-config-prettier-or-eslint-plugin-prettier/

Reply:
While I can agree with the sentiment of the article with having tools perform what they were designed for the editor integration leaves me worried.

For this change to work we'll also have to update our VSCode settings for the project and hope that others on non-VSCode editors have no difficulties with auto-formatting/linting.

I've also had issue with saveOnFormat + eslint causing an infinite loop of changes 😆 

That said it was some time ago and if we feel confident in this change we could do it!